GEON-PPANG-iOS/Presentation/BakeryDetail/Cell/WrittenReviewsCollectionViewCell.swift

```diff
@@ -128,24 +129,54 @@ final class WrittenReviewsCollectionViewCell: UICollectionViewCell {
         
         reviewTextLabel.do {
             $0.basic(font: .subHead!, color: .gbbGray400!)
-            $0.numberOfLines = 3
-            $0.adjustsFontSizeToFitWidth = true
+            $0.numberOfLines = 6
         }
     }
     
     // MARK: - Custom Method
     
-    func configureCellUI(_ data: ReviewList) {
+    func configureCellUI(_ data: ReviewList, _ labelHeight: CGFloat) {
         
         userNicknameLabel.text = data.memberNickname
         reviewDateLabel.text = data.createdAt
         reviewTextLabel.text = data.reviewText
         
         if !data.recommendKeywordList.isEmpty {
+            
             let list = data.recommendKeywordList.map {
                 $0.recommendKeywordID
             }
             reviewCategoryStackView.getChipStatus(list)
+            
+            reviewContainer.snp.remakeConstraints {
+                $0.top.equalToSuperview()
+                $0.directionalHorizontalEdges.equalToSuperview().inset(convertByWidthRatio(24))
+                $0.width.equalTo(convertByWidthRatio(327))
+                $0.height.equalTo(convertByWidthRatio(labelHeight + 111))
+            }
+            
+            reviewTextLabel.snp.remakeConstraints {
+                $0.top.equalTo(reviewCategoryStackView.snp.bottom).offset(convertByWidthRatio(16))
+                $0.directionalHorizontalEdges.equalToSuperview().inset(25)
+                $0.width.equalTo(convertByWidthRatio(277))
+                $0.height.equalTo(convertByWidthRatio(labelHeight))
+            }
+        } else {
+            reviewCategoryStackView.getNoRecommend()
+            
+            reviewContainer.snp.remakeConstraints {
+                $0.top.equalToSuperview()
+                $0.directionalHorizontalEdges.equalToSuperview().inset(convertByWidthRatio(24))
+                $0.width.equalTo(convertByWidthRatio(327))
+                $0.height.equalTo(convertByWidthRatio(labelHeight + 70))
+            }
+            
+            reviewTextLabel.snp.remakeConstraints {
+                $0.top.equalTo(profileImage.snp.bottom).offset(convertByWidthRatio(10))
+                $0.directionalHorizontalEdges.equalToSuperview().inset(25)
+                $0.width.equalTo(convertByWidthRatio(277))
+                $0.height.equalTo(convertByWidthRatio(labelHeight))
```

Comment:
p4: remakeConstraints같은 경우 중복 코드가 많은데 ,,,
```suggestion
 let isKeywordListEmpty = data.recommendKeywordList.isEmpty
        if !isKeywordListEmpty {
            let keywordIDs = data.recommendKeywordList.map { $0.recommendKeywordID }
            reviewCategoryStackView.getChipStatus(keywordIDs)
        }

        let containerHeight = convertByWidthRatio(isKeywordListEmpty ? labelHeight + 70 : labelHeight + 111)

        reviewContainer.snp.remakeConstraints {
            $0.top.equalToSuperview()
            $0.directionalHorizontalEdges.equalToSuperview().inset(convertByWidthRatio(24))
            $0.width.equalTo(convertByWidthRatio(327))
            $0.height.equalTo(containerHeight)
        }

        let textLabelOffset = convertByWidthRatio(isKeywordListEmpty ? 16 : 10)
        reviewTextLabel.snp.remakeConstraints {
            $0.top.equalTo(reviewCategoryStackView.snp.bottom).offset(textLabelOffset)
            $0.directionalHorizontalEdges.equalToSuperview().inset(25)
            $0.width.equalTo(convertByWidthRatio(277))
            $0.height.equalTo(convertByWidthRatio(labelHeight))
        }
```
삼항연산자 사용해서 나타내는건 어떤가염

Reply:
[24bce7e](https://github.com/GEON-PPANG/GEON-PPANG-iOS/pull/180/commits/24bce7e526cd2442f5197bf371a4e49b41612989)